src/libcore/platforms/nodejs.js

```diff
@@ -369,9 +369,7 @@ export default (arg: {
           } = statics[method];
           if (njsInstanciateClass) {
             m[method] = function met(...vargs) {
-              if (process.env.VERBOSE) {
-                log("libcore-call", id + "." + method, vargs);
-              }
+              log("libcore-call", id + "." + method, vargs);
```

Comment:
please either undo, or use an env. if you don't use the env, we must completely remove it 'VERBOSE_LIBCORE' didn't seem to work.